### PR TITLE
Release 0.4.2

### PR DESCRIPTION
**Pull request by jor-**
I made some small changes. Especially:
- A bug where the ordering method is not taken into account is fixed.
- The Factor class has now a (public) copy method (see issue #30).

### CODE REVIEW

File: sksparse/test_cholmod.py [hunk 1]
```diff
@@ -275,3 +278,9 @@ def test_CholmodNotPositiveDefiniteError():
     A = -sparse.eye(4).tocsc()
     f = cholesky(A)
     assert_raises(CholmodNotPositiveDefiniteError, f.L)
+
+def test_NaturalOrderingMethod():
```
Comment (anntzer) [inline sksparse/test_cholmod.py]: Should probably use underscores instead of camelcase (the one above is an actual class name).

Comment (jor-) [inline sksparse/test_cholmod.py]: You are right. I changed it.

**Comment by anntzer** [comment]
LGTM.  I'll let you self-merge in case you want to rework the history first.

**Comment by jor-** [comment]
Okay. Thank you for the review.

**Pull request merged**